Gemfile

```diff
@@ -18,6 +18,7 @@ gem "html-pipeline"
 gem "kramdown"
 gem "mdl"
 
+gem "govuk_publishing_components", "~> 23.12"
```

Comment:
What's the reason for this qualifier?

Reply:
Been having a bit of trouble getting my local setup to use a version of govuk_publishing_components higher than v21!

```
Downloading govuk_publishing_components-23.12.1 revealed dependencies not in the
  API or the lockfile (sprockets (< 4)).
  Either installing with `--full-index` or running `bundle update
  govuk_publishing_components` should fix the problem.
```

Build is currently failing with a similar message. The above was an attempt to fix the issue. I'll be making another force push shortly to hopefully fix it outright!